.github/workflows/cla_bot.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     steps:
       - name: "CLA Assistant"
         if: (github.event.comment.body == 'recheck' || github.event.comment.body == 'I have read the CLA Document and I hereby sign the CLA') || github.event_name == 'pull_request_target'
-        uses: contributor-assistant/github-action@v2
+        uses: contributor-assistant/github-action/@master
```

Comment:
can you add a comment why we should stay with `@master`?

Reply:
Done